gateway/ln-gateway/src/bin/gatewayd.rs

```diff
@@ -0,0 +1,55 @@
+use fedimint_api::task::TaskGroup;
+use fedimint_server::config::load_from_file;
+use ln_gateway::{
+    client::{GatewayClientBuilder, RocksDbFactory, StandardGatewayClientBuilder},
+    config::GatewayConfig,
+    rpc::lnrpc_client::{LnRpcClientFactory, NetworkLnRpcClientFactory},
+    utils::try_read_gateway_dir,
+    LnGateway,
+};
+use tracing::error;
+
+/**
+ * Fedimint Gateway Binary
+ *
+ * This binary runs a webserver with an API that can be used by Fedimint clients to request routing of payments
+ * through the Lightning Network. It uses a Gateway Lightning RPC client to communicate with a Lightning node.
+ */
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
```

Comment:
I think this needs to configure `tracing` in order to write any logs, right?

Reply:
Probably needs `#[tracing::instrument]` macro marker